jupyterhub/templates/NOTES.txt

```diff
@@ -1,54 +1,90 @@
-Thank you for installing JupyterHub!
+{{- $proxy_service := include "jupyterhub.proxy-public.fullname" . -}}
 
-Your release is named "{{ .Release.Name }}" and installed into the namespace "{{ .Release.Namespace }}".
+.      __                             __                   __  __            __
```

Comment:
```suggestion
{# Generated with https://patorjk.com/software/taag/#p=display&h=0&f=Slant&t=JupyterHub #}
.      __                             __                   __  __            __
```

Reply:
@yuvipanda I made this to be a Helm template comment syntax wise, was that your intention? I think it is what makes most sense for this.